docs/src/userguide/language_basics.rst

```diff
@@ -247,8 +247,61 @@ Types
 
 The Cython language uses the normal C syntax for C types, including pointers.  It provides
 all the standard C types, namely ``char``, ``short``, ``int``, ``long``,
-``long long`` as well as their ``unsigned`` versions,
-e.g. ``unsigned int`` (``cython.uint`` in Python code).
+``long long`` as well as their ``/unsigned`` versions,
```

Comment:
Is the `/` in front of `unsigned` doing anything specific.
?

Reply:
It's a typo. Let me fix that.